tls/s2n_config.c

```diff
@@ -68,19 +69,22 @@ static int s2n_config_setup_default(struct s2n_config *config)
 {
     GUARD(s2n_config_set_cipher_preferences(config, "default"));
     GUARD(s2n_config_set_signature_preferences(config, "default"));
+    GUARD(s2n_config_set_ecc_preferences(config,"default"));
     return S2N_SUCCESS;
 }
 
 static int s2n_config_setup_tls13(struct s2n_config *config)
 {
     GUARD(s2n_config_set_cipher_preferences(config, "default_tls13"));
     GUARD(s2n_config_set_signature_preferences(config, "default_tls13"));
+    GUARD(s2n_config_set_ecc_preferences(config,"default_tls13"));
     return S2N_SUCCESS;
 }
 
 static int s2n_config_setup_fips(struct s2n_config *config)
 {
     GUARD(s2n_config_set_cipher_preferences(config, "default_fips"));
+    GUARD(s2n_config_set_ecc_preferences(config,"default_tls13"));
```

Comment:
Why tls13 here and not a fips default?

Reply:
will add a `default_fips`

Reply:
You don't need a new default -- the existing "default" should be fine :)